runner/runner.go

```diff
@@ -902,6 +961,20 @@ func (r *Runner) RunEnumeration() {
 	wgoutput.Wait()
 }
 
+func openOrCreateFile(resume bool, filename string) *os.File {
```

Comment:
This looks a good candidate helper for `utils` package, what do you think?

Reply:
I agree! I created an issue for it https://github.com/projectdiscovery/utils/issues/205.